administration/src/types/assets.d.ts

```diff
@@ -2,3 +2,8 @@ declare module '*.pdf' {
   const content: string
   export default content
 }
+
+declare module '*.png' {
```

Comment:
please undo.

Reply:
do you mean change it to svg, or there is a better way to import images?

Reply:
yes change to svg :+1: